src/main/java/ua/lviv/iot/parkingServer/model/Record.kt

```diff
@@ -0,0 +1,6 @@
+package ua.lviv.iot.parkingServer.model
+
+abstract class Record {
+    abstract fun getHeaders(): String
+    abstract fun toCSV(): String
+}
```

Comment:
no state, no abstract class :)

Move to interface

Reply:
✅